api/HTMLMediaElement.json

```diff
@@ -2084,20 +2084,30 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLMediaElement/srcObject",
           "support": {
             "chrome": {
-              "version_added": "52"
+              "version_added": "52",
+              "partial_implementation": true,
+              "notes": "Only MediaStream is supported"
```

Comment:
The note should end with a period, since it's a complete sentence, and "MediaStream" needs to be inside a `<code>` element.

Since it's being changed anyway, might also be worth rewording to something like `"Currently only supports <code>MediaStream</code> objects."` but that's not mandatory to make  this good to go...